tests/cuda/CMakeLists.txt

```diff
@@ -14,6 +14,27 @@ include( vecmem-compiler-options-cuda )
 # External dependency/dependencies.
 find_package( CUDAToolkit REQUIRED )
 
+add_library(
+   vecmem_testing_cuda_main
+   SHARED
+   test_cuda_main.cu
+)
+
+target_link_libraries(
+   vecmem_testing_cuda_main
+   PUBLIC
+   CUDA::cudart
+   gtest
```

Comment:
This should be `GTest::gtest`. (`gtest` only works if GoogleTest is built as part of VecMem. When taking GoogleTest from "the system", it doesn't.)